docker-compose.i7-13700k.yaml

```
@@ -0,0 +1,152 @@
+services:
+  neo4j:
+    image: neo4j:5.15-community
+    container_name: crawler-neo4j-13700k
+    restart: unless-stopped
+    ports:
+      - "7474:7474"
+      - "7687:7687"
+    environment:
+      # Authentication
+      - NEO4J_AUTH=neo4j/password
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)